Refresh installation guide, with focus on bundled app instructions (#958)
napari/docs: docs/getting_started/installation.md

Reviewer: no longer really relevant.
```suggestion
If you prefer to manage packages with conda, napari is available on the
conda-forge channel. You can install it with:
```

# Install napari

This guide will teach you how to do a clean install of **napari** and launch the viewer.
`````{grid} 1 2 2 2

> If you're already familiar with Python, napari can be installed from PyPI or conda-forge using your favorite virtual environment and package manager. Otherwise, keep reading for instructions on how to install napari as a Python package.

Reviewer: I'm partial to this sentence about being able to use your favorite virtual environment and package manager. We do want the docs to be opinionated, but if someone is more advanced and has their own preferences they can go with that!

## Install as an application

> napari can be installed as a standalone application on macOS, Windows, and Linux — no Python knowledge required. This is the simplest way to get started and works best if you want to use napari as a standalone GUI app. Note that some plugins may not be available without a full Python environment.

Reviewer: How about this phrasing?
```suggestion
napari can be installed as a standalone application on macOS, Windows, and Linux — no Python knowledge required. This is the simplest way to get started and works best if you want to use napari as a standalone GUI app. Note that some plugins may not be fully supported by just the GUI.
```
(very apropos I got an email today from a user confused by something from plugin docs which assumed CLI usage -- we got it working via the console, so it was a nothing burger)
